snippets/card-product.liquid

```diff
@@ -200,6 +200,11 @@
             {%- endif -%}
 
             {% render 'price', product: card_product, price_class: '' %}
+            {%- if card_product.quantity_price_breaks_configured? -%}
+              <div class="card__information--volume-pricing-note">
```

Comment:
Regarding the selector naming, why is this selector a modifier of `card__information`? What is the relationship between this element and the other `card__information` elements?

Reply:
If we're using BEM convention here, a modifier (`card__information--volume-pricing-note`) should not be applied to a child of `card__information`